site/app/templates/forum/ShowCategories.twig

```diff
@@ -8,38 +8,37 @@
         "show_filter": false
     } %}
 
-    <h3 style="margin-bottom:10px;"> Manage Categories </h3>
-
-    <div id="manage_categories_view" style="margin: 0 auto; max-width: 750px;">
+    <div id="manage_categories_view" style="margin: 0 auto; max-width: 750px; padding-bottom: 20px">
```

Comment:
ID's and classes should use dashes not underscores
https://submitty.org/developer/coding_style_guide/css#styling

Reply:
Ok, I didn't write it, but I'll change this to fit the style guide :)

Reply:
Oh my bad, the github diff made it look like it was a new addition, thanks for fixing it anyways :)

Reply:
I didn't see any other usages of this id, so the modifications were needed only for this line in this file